ncm-metaconfig/src/main/pan/components/metaconfig/schema.pan

```diff
@@ -15,7 +15,6 @@ type ${project.artifactId}_config =  {
      'mode' : long = 0644
      'owner' : string = 'root'
      'group' : string = 'root'
-     'daemon' ? string[]
```

Comment:
No that's cool, I didn't realise anyone did such strange things with components :scream:, I'll bump this to 14.6 and open a new PR for the schema change.